docs/cns.md

```diff
@@ -0,0 +1,110 @@
+# Chain Name Service Draft
+
+The goal of this module is to be a simple registry with the ability to transfer ownership. Auctions and other mechanisms are out of scope and can be implemented alongside or on top of this system (perhaps, in a custodial way).
+
+# State
+
+## Chain name
+
+Chain name contains information about the chain, most importantly, chain ID used to uniquly identify the chain and a list of peer nodes.
+
+Key: `chain-[chainID] -> Chain Name` 
+
+```go
+type ChainName struct {
+	ChainID string
+	Owner sdk.AccAddress
```

Comment:
Given these might eventually be controlled over IBC, this should be a string. The counterparty chains may not have a notion of sdk.AccAddress. The owner may be a channel/port pair.

Reply:
If it were controlled over IBC it would still use an Interchain Account which is represented by `sdk.AccAddress`, right? Is there another format where an IBC channel/port pair would be preferable?